submissions/2024-04-02-xtalpi-20-percent-fragments-torsiondrive-v1.0/README.md

```diff
@@ -0,0 +1,64 @@
+# XtalPi 20-percent Fragments TorsiondriveDataset v1.0
+
+## Description
+
+A dataset containing representative fragments shared by XtalPi
+used in fitting the XFF force field
+(DOI: [10.1021/acs.jctc.3c00920](https://doi.org/10.1021/acs.jctc.3c00920)). This is a continuation of the
+[XtalPi Shared Fragments TorsiondriveDataset v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-01-30-xtalpi-shared-fragments-torsiondrive-v1.0) dataset.
+Where this previous dataset contained ~3% the original XFF fitting dataset,
+this dataset contains an additional 20%. 
+Conformers are the post-optimization geometries shared by XtalPi.
+Each conformer will be converged according to the 'GAU_LOOSE' criteria.
+
+MOL2 files were first converted into a PyArrow dataset
+using the Python script included.
+This dataset is read in the Jupyter notebook to create the submission.
+
+### Change log
+* v1.0 Initial submission
+
+## General information
+
+* Date: 2024-04-02
+* Class:  OpenFF TorsionDrive Dataset
+* Purpose: Torsion scans of small molecules assembled from ChEMBL fragments
+* Name: XtalPi 20-percent Fragments TorsiondriveDataset v1.0
+* Number of unique molecules: 8737
+* Number of conformers (min, mean, max): 1 2.5 8
+* Molecular weight (min, mean, max): 55.87 195.69 387.45
+* Charges: -2, -1, 0, 1, 2
+* Dataset submitter: Lily Wang
+* Dataset generator: Lily Wang
+
+
+## QCSubmit generation pipeline
+
+* `generate-dataset.ipynb`: This notebook shows how the dataset was prepared from the input files.
+
+
+## QCSubmit Manifest
+
+* `dataset.json.bz2`: Compressed dataset ready for submission
+* `dataset.pdf.tar.gz`: Visualization of dataaset molecules, compressed
```

Comment:
Same typo as opt

Reply:
Thanks for the review, @ntBre -- fixed the typo + GAU_LOOSE issue!